REQUIRE

```diff
@@ -1,3 +1,7 @@
 julia 0.5-
 CoordinateTransformations 0.3.0
-StaticArrays 0.0.4
+
+# Enforce that:
+# - Julia 0.6 only using StaticArrays 0.5.0 and up
+# - Julia 0.5 uses StaticArrays between 0.0.4 and 0.4.0-
+StaticArrays 0.0.4 0.4.0- 0.5.0
```

Comment:
So, does this strange looking version specification allows `Geodesy` version resolution to work on both 0.5 and 0.6?

That's great if it works!  I had an uneasy but vague memory that it was more complicated than this... @andyferris do you remember the issue where we discussed this at length?  There's https://github.com/JuliaArrays/StaticArrays.jl/issues/152 , but that doesn't seem to be it.